docs/UserGuide.md

```diff
@@ -232,6 +230,24 @@ Examples:
 * `addEvent n/Shoe Sale 30% d/30-05-2022 t/11:00 p/Discount on all shoes for up to 30%`
 * `addEvent n/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
 
+### Editing an event : `editEvent`
+
+Edits an existing event in the application.
+
+Format: `editEvent INDEX [e/EVENT_TITLE] [d/DATE] [t/TIME] [p/PURPOSE]`
+
+- Edits the event at the specified `INDEX`. The index refers to the index number shown in the displayed event list.
+  The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the event list index.
+- At least one of the optional fields must be provided.
```

Comment:
```suggestion
```
I think this can be removed as well?

Reply:
Ah, this optional field doesn't refer to DOB/ Gender etc coz in EditEvents (and EditPersons for that matter) all fields are optional. So this condition "at least one of the optional fields must be provided" must hold true for this command to work, meaning at least one field must follow after the command + index.